ui/packages/consul-ui/app/components/topology-metrics/stats/index.hbs

```diff
@@ -15,7 +15,7 @@
   />
 {{/if}}
 
-<div class="stats">
+<div class="topology-metrics-stats" data-test-stats>
```

Comment:
I took the opportunity to update the class name here.

Reply:
Awesome, thanks for doing this! Although you got me wondering whether we should use `data-test-topology-metrics-stats` instead of just `data-test-stats`?

Reply:
Updated and pushed

Reply:
No action here (I'm 100% fine with the update), but this gets into the redundancy (i.e. we change one and we have to change the other to be the same) vs avoid non `data-test-*` selectors for testing - just something that's still an ongoing thing in my head, thought I'd mention that here.